Makefile

```diff
@@ -19,7 +29,7 @@ else
 	CMAKE_GENERATOR ?= "Unix Makefiles"
 	MAKE = make
 	# Use 8 processes in parallel for the usual make
-	MAKE_ARGS = -j8
+	MAKE_ARGS = -j$(NPROCS)
```

Comment:
Please remove the comment above since it is no longer true.

Reply:
Removed!